### DISCUSSION TITLE
LiveResize dysków dla działających VM

### DISCUSSION THREAD

**Discussion by MarekSuwart** [AzureCommunityPL/group#455] [Pomoc]
Cześć,
jakiś czas temu M$ wprowadził nowy feature "LiveResize" dla dysków podpiętych pod działające maszynki.  Jeden z artykułów w tym temacie https://blog.hametbenoit.info/2022/07/25/azure-you-can-now-expand-your-data-drive-without-downtime-preview/ .
Zgodnie z procedurką zarejestrowalem feature  ( ma status registered) ale wciaz nie mam opcji aby dodac przestzreni do dysku podpietego do dzialajacej maszynki, czekam juz z godzine i nadal nic :) . Czy ktos  z was przerabiał ten temat ?
Dzieki

**Comment by sbugalski** [comment]
Tzn nie widzisz w systemie?
A próbowałeś zrobić resize partycji?

**Reply by MarekSuwart** [reply]
z portalu\cli nie moge , tak jakby to nie zatrybiło, moze trzeba dluzej poczekac ?:)

**Comment by Rafal-Bielecki** [comment]
Feature już jest ok, nawet napisałem mały skrypcik który rozszerza zarówno w chmurce jak i w OS dany dysk :)
